src/components/shows/heroBanner.module.css

```diff
@@ -0,0 +1,75 @@
+.hero {
```

Comment:
If you can be arsed, CSS styles for each class/ID should be ordered alphabetically.

Reply:
Also done :)